functionnal_tests/cypress/integration/app_html-document/operation_allcontent_delete_html-document.js

```diff
@@ -17,7 +17,7 @@ describe('operation :: workspace > delete > html-document', function () {
     cy.get('.cardPopup__container .createcontent .createcontent__form__input').should('have.attr', 'placeholder')
     cy.get('.cardPopup__container .createcontent .createcontent__form__input').type(titre1)
     cy.get('.cardPopup__container .createcontent .createcontent__form__input').should('have.attr', 'value', titre1)
-    cy.get('.cardPopup__container .createcontent .createcontent__form__button.highlightBtn').click()
+    cy.get('[data-cy=createcontent__form__button]').click()
```

Comment:
My feeling is

- `cy.get('.cardPopup__container .createcontent .createcontent__form__button').click() `

is easier to understand than

- `cy.get('[data-cy=createcontent__form__button]').click()`

Reply:
```
cy.get('.cardPopup__container .createcontent .createcontent__form__button').click() 
```
Uses css classes that can change and break the test for no good reason (that why I had to fix this test), `data-cy` sometimes are less descriptive but they don't change.

Reply:
Done

Reply:
actually, i'm not sure using a non-evolving data-cy property will be better than breaking changes caused by css classes requiring test updates... in a few chagnes, data-cy property name will have no logic related to css classes and maintaining the code will be a pain.

The right solution is to have semantic structure in the user interface.